### ISSUE TITLE
Weird doc string in wirepattern

### ISSUE THREAD

**Issue by ensonic** [noflo/noflo#281]
https://github.com/noflo/noflo/blob/master/src/lib/Helpers.coffee#L75-L77
This sentence is hard to parse. I checked git blame, but it was originally commited as is.

**Comment by trustmaster** [comment]
This seems to come from the early days when WirePattern could be described in one weird sentence. Do you think we could just drop it?

**Comment by ensonic** [comment]
+1 from removing. The behaviour wrt config.group is documented above.
